examples/auth0/pages/index.js

```diff
@@ -1,38 +1,21 @@
-import React from 'react'
-
 import Layout from '../components/layout'
 import { useFetchUser } from '../lib/user'
 
 function Home() {
-  const { user, loading } = useFetchUser()
+  const { user } = useFetchUser()
+  if (user) { window.location = "/profile"; }
```

Comment:
Why is the main route no longer accessible to the user? I don't see any reason to hide it. This was a good example of how the user is handled in the browser, can we undo this? Also, `Router.push` is a better option here because the navigation is to another page.